examples/backends/minimal/src/minimal.cc

```diff
@@ -307,9 +280,9 @@ TRITONBACKEND_ModelInstanceExecute(
   // and the corresponding TRITONBACKEND_Response object within
   // 'responses' is set to nullptr to indicate that the
   // request/response has already been handled an no futher processing
-  // should be performed for that
-  // request. RESPOND_ALL_AND_RETURN_IF_ERROR,
-  // RESPOND_AND_SET_NULL_IF_ERROR, and
+  // should be performed for that request. Even if all responses fail,
+  // the backend still allows execution to flow to the end of the
```

Comment:
Is this a general advice? If all responses have been sent; as a part of (RESPOND_ALL_AND_SET_NULL_IF_ERROR/RESPOND_ALL_AND_RETURN_IF_ERROR) doesn't it make sense to directly return? I understand we skip on accounting these in our inferstatistics as failed runs.. But that it how many of our backends are designed currently..

Reply:
Should I replace RESPOND_ALL_AND_RETURN_IF_ERROR from these backends?

Reply:
Yes, this is general advice (but of course other ways of doing it are possible). It is an error to skip accounting for the failures and that is why we have this advice... but you are right that we may be doing it wrong in several backends.